src/main/java/org/folio/circulation/support/http/server/ValidationError.java

```diff
@@ -37,9 +63,15 @@ public JsonObject toJson() {
             .put("value", parameters.get(key)))
         .collect(Collectors.toList()));
 
-    return new JsonObject()
+    JsonObject result = new JsonObject()
       .put("message", message)
       .put("parameters", mappedParameters);
+
+    if (code != null) {
+      JsonPropertyWriter.write(result, "code", code.toString());
+    }
```

Comment:
```suggestion
      JsonPropertyWriter.write(result, "code", code.toString());
```
because JsonPropertyWriter.write  has already check for not null